setup.cfg.template

```diff
@@ -13,7 +13,7 @@
 # set this to True.  It will download and build a specific version of
 # FreeType, and then use that to build the ft2font extension.  This
 # ensures that test images are exactly reproducible.
-#local_freetype = False
```

Comment:
Please do not commit this change.

The same effect can be achived by

```
export MPLLOCALFREETYPE=1
```

in the enviroment you are building in.